Makefile

```diff
@@ -240,6 +240,12 @@ ifeq ($(compatversion),y)
 	cp $(COMPAT_DIR)/ChangeLog $(DESTDIR)$(docdir)/ChangeLog-compat
 	cp $(COMPAT_DIR)/NEWS $(DESTDIR)$(docdir)/NEWS-compat
 endif
+endif
+ifeq ($(vimplugin),y)
+	mkdir -p $(DESTDIR)$(vimplugindir)
+	install -D contrib/vim/ftdetect/modulefile.vim  $(DESTDIR)$(vimplugindir)/ftdetect/modulefile.vim
+	install -D contrib/vim/ftplugin//modulefile.vim $(DESTDIR)$(vimplugindir)/ftplugin/modulefile.vim
+	install -D contrib/vim/syntax/modulefile.vim    $(DESTDIR)$(vimplugindir)/syntax/modulefile.vim
```

Comment:
I would prefer not relying on an additional tool to install and having these files and dirs installed the same way other files here does (mkdir+cp)